docusaurus/docs/extensions/catalog.mdx

```diff
@@ -0,0 +1,43 @@
+# Extensions Catalog
+
+import Catalog from '../../plugins/catalog/Catalog.tsx'
+
+For information on adding your extension to this page, see here. For information of developing your own extension, check out the [developer documentation](http://localhost:3000/extensions/next/extensions-getting-started).
```

Comment:
should the `here` in `see here` a link? If not, then the whole sentence feels a bit odd since we are telling people to see this page to add extensions to the page... Doesn't it feel a bit odd/weird? 🤔  can't explain it, but something sounds off to me 😛 

Feel free to ignore if you disagree 🙏